README.md

```diff
@@ -136,7 +136,7 @@ Add to mix.exs:
 
 ```elixir
 defp deps do
-  [{:ecto_soft_delete, "~> 2.0"}]
+  [{:ecto_soft_delete, "~> 2.0.5"}]
```

Comment:
I don' think this change is really necessary since `"~> 2.0"` means `>= 2.0.0 and < 3.0.0`, but it's probably OK